src/main/java/github/grace000/erversay/RequestParser.java

```diff
@@ -0,0 +1,13 @@
+package github.grace000.erversay;
+
+public class RequestParser {
+    public String[] parse(String requestLine) {
+        if (requestLine == null || requestLine.isEmpty()) {
+            System.out.println("REQUEST LINE CANNOT BE PARSED");
+            requestLine = "/GET NOT_FOUND HTTP/1.1";
```

Comment:
Can you elaborate on why you chose to fall back to an arbitrary string here?

Reply:
I figured that empty requests are not valid requests and invalid requests should have "unknown" responses. In my case, I'm getting empty requests for favicons, but even though I'm not parsing those requests, the client is not being disturbed. If the client sends a favicon request, the browser remains with whichever resource it had prior to those additional empty requests.